.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: node_js
 cache: yarn
+install:
+- npm install -g ng-cli
```

Comment:
This was needed for tests to start running.

Reply:
Actually noticed although it made the build pass green the tests are not running ...